### PR TITLE
README.rst: Replace broken links

### PR DESCRIPTION
**Pull request by ashwani99**
Fixes https://github.com/coala/coala-bears/issues/1159

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ The number of bears grows every day! If you want to see any particular
 functionality be sure to submit an issue, but please read the `Getting Involved`_
 section before doing so.
 
-You can read more at our `documentation <http://coala.readthedocs.io/en/latest/Developers/Writing_Bears.html>`__.
+You can read more at our `documentation <http://api.coala.io/en/latest/Developers/Writing_Bears.html>`__.
```
Comment (sils) [inline README.rst]: can you use coala.io/writingbears ?

File: README.rst [hunk 2]
```diff
@@ -138,7 +138,7 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.readthedocs.io/en/latest/Developers/Newcomers_Guide.html>`__
+`newcomer guide <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`__
```
Comment (sils) [inline README.rst]: coala.io/newcomer

**Comment by ashwani99** [comment]
I have updated the links accordingly.

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1160/commits/fab2e8af9ae961728e2853d51fa4457797717823

**Comment by sils** [comment]
commits need squashing!

**Pull request closed without merging**